src/components/header/Header.js

```diff
@@ -4,12 +4,31 @@ import { Link } from 'react-router-dom'
 export const Header = () => {
   return (
     <header>
-      <h1>Pawty Trainer</h1>
+      <section className='header-container'>
+        <h1 className='title'>Pawty Trainer</h1>
+        {/* <p className='material-icons'>pets</p> */}
+      </section>
       <nav>
-        <Link to ='/'>Dashboard</Link>
-        <Link to ='/create'>Add Dog</Link>
-        <Link to ='/add_event'>Add Event</Link>
-        <Link to ='/calendar'>Calendar</Link>
+        <Link 
```

Comment:
This looks wayyy less readable to me. Not necessary because those lines aren't exceeding 80 characters.

Reply:
sorry, i was playing around with adding a classname. ill change it back.